data/repository/src/commonMain/kotlin/social/androiddev/common/AuthenticationRepositoryImpl.kt

```diff
@@ -0,0 +1,32 @@
+package social.androiddev.common
+
+import social.androiddev.common.network.MastodonApi
+import social.androiddev.domain.authentication.repository.AuthenticationRepository
+
+class AuthenticationRepositoryImpl(
```

Comment:
Same comment again re `internal` and a static creation method. We should do what we can to minimize API surfaces.

Reply:
https://github.com/AndroidDev-social/MastodonCompose/pull/68#discussion_r1024762246